sdk/eventhub/eventhubs-checkpointstore-table/src/tableCheckpointStore.ts

```diff
@@ -22,14 +66,37 @@ export class TableCheckpointStore implements CheckpointStore {
    *  - `tracingOptions`: Options for configuring tracing.
    * @returns Partition ownership details of all the partitions that have had an owner.
    */
-  async listOwnership(): /*
-        fullyQualifiedNamespace: string,
-        eventHubName: string,
-        consumerGroup: string,
-        options: OperationOptions = {}
-        */
-  Promise<PartitionOwnership[]> {
-    throw new Error("not implemented");
+  async listOwnership(
+    fullyQualifiedNamespace: string,
+    eventHubName: string,
+    consumerGroup: string
+  ): Promise<PartitionOwnership[]> {
+    const partitionKey = `${fullyQualifiedNamespace} ${eventHubName} ${consumerGroup} Ownership`;
+    const partitionOwnershipArray: PartitionOwnership[] = [];
+    const entitiesIter = this._tableClient.listEntities<PartitionOwnershipEntity>({
+      queryOptions: { filter: odata`PartitionKey eq ${partitionKey}` }
+    });
+    try {
+      for await (const entity of entitiesIter) {
+        const partitionOwnership: PartitionOwnership = {
+          fullyQualifiedNamespace,
+          eventHubName,
+          consumerGroup,
+          ownerId: entity.ownerid,
+          partitionId: entity.rowKey,
+          lastModifiedTimeInMs: await this._toMillisecs(entity.timestamp),
```

Comment:
It looks like `entity.timestamp` should always be defined since we aren't using a `select` filter in the query.

That said...without this field the ownership won't work. I think it might be worth throwing an error before you create the `partitionOwnership` variable if it's missing. Something like:

```ts
if (!entity.timestamp) {
  throw new Error(`Unable to retrieve timestamp from partitionKey "${partitionKey}", rowKey "${entity.rowKey}"`);
}
```

Then I think you can get rid of the `_toMillisecs` method and just do `new Date(entity.timestamp).getTime()`